### PR TITLE
docs: wrap tokens tables in shortcode

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Wrapped token tables in `{% tokensTable %}` shortcode

## Testing Instructions

1. View [deploy preview](https://deploy-preview-1077--red-hat-design-system.netlify.app/)

## Notes to Reviewers

All tables that included css hex codes should now be hidden.  Any associated headers with those tables should also be wrapped in the shortcode so they are no longer visible.

### CODE REVIEW

File: docs/_plugins/shortcodes/tokensTable.cjs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPairedShortcode('tokensTable',
+    /**
+     * Tokens Table
+     * Display a table rows with token usage
+     */
+    function(content) {
```
Comment (bennypowers) [inline docs/_plugins/shortcodes/tokensTable.cjs:7]:
```suggestion
    function(content, options) {
```

include: string
exclude: string
only: 'tokens'|'cssProps'
types: string
element: string

I _think_ that would be enough? but if you think this is jumping the gun, let's defer until we talk it over with design

File: elements/rh-footer/docs/10-style.md [hunk 2]
```diff
@@ -47,6 +47,8 @@ theme.
 
 ### Website-specific footer
```
Comment (nikkimk) [inline elements/rh-footer/docs/10-style.md:48]: Some of these headings have nothing but token tables below them. We might want to move those headings into the shortcode as well.

Comment (zeroedin) [reply elements/rh-footer/docs/10-style.md:48]: Example: https://deploy-preview-1077--red-hat-design-system.netlify.app/elements/footer/style/#website-specific-footer

These headers only contained token tables

File: docs/_plugins/shortcodes/tokensTable.cjs (hunk 1 above)
Comment (zeroedin) [reply docs/_plugins/shortcodes/tokensTable.cjs:7]: Let's defer this till we know more what we want this shortcode to do.

**Review by coreyvickery — APPROVED** [review]
Looks good, thanks.

**Pull request merged**